README.md

```diff
@@ -27,60 +74,106 @@ mutation SignInMutation($input: signInInput!) {
 
-### With a cookie
+```bash
+$ curl 'https://api.sorare.com/graphql' \
+-H 'content-type: application/json' \
+-H 'x-csrf-token: <TheSameCSRFTokenValue>' \
+-H 'cookie: _sorare_session_id=<TheSameCookieValue>' \
+-d '{
+  "operationName": "CurrentUserQuery",
+  "query": "query CurrentUserQuery { currentUser { slug email } }"
+}'
 
-To use a cookie authentication you need to store the new `_sorare_session_id` and `X-CSRF-TOKEN` and pass it to all next requests.
+{"data":{"currentUser":{"slug":"<YourSlug>","email":"<YourEmail>"}}}
+```
+
+This is exactly the authentication mechanism that happens while browsing the [sorare.com](https://sorare.com) website.
+
+### GraphQL `signIn` mutation & JWT tokens
 
-### With a JWT token
+A better way than cookies to stay authenticated is to rely on [JWT tokens](https://jwt.io/).
 
-To authenticate by JWT you need to request a valid JWT token in the `signIn` mutation :
+We provide JWT tokens within the `signIn` mutation. They can be retrieve using the following mutation:
 
 ```gql
-  mutation SignInMutation($input: signInInput!) {
-    signIn(input: $input) {
-      currentUser {
-        slug
-        jwtToken(aud: "YOUR_AUD") {
-          token
-          expiredAt
-        }
+mutation SignInMutation($input: signInInput!) {
+  signIn(input: $input) {
+    currentUser {
+      slug
+      jwtToken(aud: "<YourAud>") {
+        token
+        expiredAt
       }
-      ...
     }
   }
+}
 ```
 
-`YOUR_AUD` is a mandatory string parameter that helps identifying your app. Also, tokens generated with an `aud` are scoped to this `aud`. We recommend to use an `aud` reflecting the name of your app - like `myrealappname` - to make it easier to debug & track.
+`<YourAud>` is a mandatory _string_ parameter that helps (us & you) identifying your app. We recommend to use an `aud` reflecting the name of your app - like `myappname` - to make it easier to debug & track.
 
-You shall then pass it to all next requests through the following headers:
+```bash
+$ curl 'https://api.sorare.com/graphql' \
+-H 'content-type: application/json' \
+-H 'x-csrf-token: <YourCSRFTokenValue>' \
+-H 'cookie: _sorare_session_id=<YourCookieValue>' \
+-d '{
+  "operationName": "SignInMutation",
+  "variables": { "input": { "email": "<YourEmail>", "password": "<YourHashPassword>" } },
+  "query": "mutation SignInMutation($input: signInInput!) { signIn(input: $input) { currentUser { slug email jwtToken(aud: \"<YourAud>\") { token expiredAt } } errors { message } } }"
+}'
+
+{"data":{"signIn":{"currentUser":{"slug":"<YourSlug>","email":"<YourEmail>","jwtToken":{"token":"<YOUR JWT TOKEN>","expiredAt":"..."}},"errors":[]}}}
 ```
-Authorization: Bearer YOUR_TOKEN
-JWT-AUD: YOUR_AUD
+
+You shall then pass the token with an `Authorization` header alongside a `JWT-AUD` header to all next API requests instead of the `cookie` and the `x-csrf-token` headers:
```

Comment:
_Once you've retrieved a JWT token, you no longer need to use the `cookie` and `x-csrf-token` headers on your API requests, instead use an `Authorization` header containing the JWT token and a `JWT-AUD` header containing the `aud` string:_

Reply:
(realistically you'll never need a cookie/x-csrf-token with JWT)